src/utils/validators.tsx

```diff
@@ -104,6 +104,8 @@ const badStatusHandler = (familyCode: HTTPFamilyResponseStatusCode) =>
           utils.url.redirectWithOutcome(
             actualCode === 401
               ? OUTCOME_ROUTE.AUTH_ERROR
+              : actualCode === 422
```

Comment:
Just a reminder: this code is executed for most of the api calls and not only the ones that handle the payment manager flow. In case of a 422 in any of this api calls the redirect is going to contain a 15 outcome code

Reply:
@infantesimone I think this behaviour, also applied in other domain, can be considered desired!